ui-testing/src/main/java/com/qreal/wmp/uitesting/dia/scene/elements/SceneElementImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.qreal.wmp.uitesting.dia.scene.elements;
+
+import com.codeborne.selenide.SelenideElement;
+import com.qreal.wmp.uitesting.dia.scene.Coordinate;
+import com.qreal.wmp.uitesting.exceptions.ElementNotOnTheSceneException;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class SceneElementImpl implements SceneElement {
+    
+    private By by;
```

Comment:
This looks awesome, but I honestly have no clue what it means :)
A comment would be nice.